website/content/docs/upgrading/upgrade-to-1.10.x.mdx

```diff
@@ -62,17 +62,35 @@ OIDC provider system to reduce configuration steps and enhance usability.
 The following built-in resources are included in each Vault namespace starting with Vault
 1.10:
 
- - A "default" OIDC provider that's usable by all client applications
- - A "default" key for signing and verification of ID tokens
- - An "allow_all" assignment which authorizes all Vault entities to authenticate via a
-   client application
+- A "default" OIDC provider that's usable by all client applications
+- A "default" key for signing and verification of ID tokens
+- An "allow_all" assignment which authorizes all Vault entities to authenticate via a
+  client application
 
 If you created an [OIDC provider](/api-docs/secret/identity/oidc-provider#create-or-update-a-provider)
 with the name "default", [key](/api-docs/secret/identity/tokens#create-a-named-key) with the
 name "default", or [assignment](/api-docs/secret/identity/oidc-provider#create-or-update-an-assignment)
-with the name "allow_all" using the Vault 1.9 tech preview, the installation of these built-in
-resources will be skipped. We _strongly_ recommend that you delete any resources that have
+with the name "allow*all" using the Vault 1.9 tech preview, the installation of these built-in
+resources will be skipped. We \_strongly* recommend that you delete any resources that have
 naming collisions before upgrading to Vault 1.10. Failing to delete resources with naming
 collisions could result unexpected default behavior. Additionally, we recommend reading the
 corresponding details in the OIDC provider [concepts](/docs/concepts/oidc-provider) document
 to understand how the built-in resources are used in the system.
+
+## Single Vault follower restart causes election even with established quorum
+
+We now support Server Side Consistent Tokens (See [Replication](/docs/configuration/replication), [Vault Eventual Consistency](/docs/enterprise/consistency), and [Upgrade to 1.10](/docs/upgrading/upgrade-to-1.10.x).), which introduces a new token format that can only be used on nodes of 1.10 or higher version. This new format is enabled by default upon upgrading to the new version. Old format tokens can be read by Vault 1.10, but the new format Vault 1.10 tokens cannot be read by older Vault versions.
```

Comment:
Ideally, these notes on this page and the release notes would be @include directives - you can find some examples on the 1.8.x upgrade page. That lets us make an update in only one place if we need to update the known issue announcement with fix version information.

Reply:
OK Will fix this in a separate PR. Right now, I have to get this merged to fix some broken links.